lib/suspenders/app_builder.rb

```diff
@@ -1,6 +1,11 @@
+require "forwardable"
+
 module Suspenders
   class AppBuilder < Rails::AppBuilder
     include Suspenders::Actions
+    extend Forwardable
```

Comment:
Simple Delegator only allows delegation to one object, which you define in
the initialization. Right now we only have one adapter, but we can have
more (at least GitHub), so Forwardable gives us that flexibility.

On Fri, 30 Oct 2015 at 19:09, Tute Costa <notifications@github.com> wrote:

> In lib/suspenders/app_builder.rb
> <https://github.com/thoughtbot/suspenders/pull/646#discussion_r43541161>:
>
> >  module Suspenders
> >    class AppBuilder < Rails::AppBuilder
> >      include Suspenders::Actions
> > +    extend Forwardable
>
> Do we need Forwardable? Does SimpleDelegator achieve the same result? How
> do they differ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/suspenders/pull/646/files#r43541161>.
>